src/plugin/duration/index.js

```diff
@@ -257,6 +257,15 @@ class Duration {
   asYears() { return this.as('years') }
 }
 
+const manipulateDuration = (date, duration, k) =>
+  date.add(duration.years() * k, 'year')
+    .add(duration.months() * k, 'month')
+    .add(duration.days() * k, 'day')
+    .add(duration.hours() * k, 'hour')
+    .add(duration.minutes() * k, 'minute')
+    .add(duration.seconds() * k, 'second')
+    .add(duration.milliseconds() * k, 'millisecond')
```

Comment:
we can replace 'second' with 's' according to https://github.com/iamkun/dayjs/blob/dev/src/utils.js#L31

to save some bytes